website/build.gradle

```diff
@@ -115,9 +115,39 @@ task buildCodeSamples(type: Exec) {
               "${->startDockerContainer.containerId()}", 'yarn', 'build_code_samples'
 }
 
+task cloneRepository(type: Exec) {
+  def branch = System.getenv('ghBranch') ?  System.getenv('ghBranch'):
+          grgit ? grgit.branch.current().getName(): 'master'
+  def repositoryUrl = System.getenv('repositoryUrl') ? "https://github.com/${System.getenv('repositoryUrl')}" :
```

Comment:
Will this ever not just be apache/beam?

Reply:
It will be different when it is run from a fork, but we can leave it as just apache/beam

Reply:
I think leaving it as is will be fine